ansible/playbooks/dev/main.yml

```diff
@@ -0,0 +1,5 @@
+- name: Install docker
+  hosts: baka
+  become: true
+  roles:
+    - docker
```

Comment:
idk something like that would save you from copy-paste ![](https://steamcommunity-a.akamaihd.net/economy/emoticon/:ohh_yeah:)
```yaml
- hosts: baka
  roles:
      - geerlingguy.docker
```